.github/workflows/web3_compatible.yml

```diff
@@ -4,215 +4,134 @@ on:
   push:
     branches:
       - main
-  pull_request:
```

Comment:
Since this workflow is `time consuming type`, I think it's better to use `merge_group` event instead of too many `pull_request` event.

Reply:
Push main is also not necessary if using merge queue.

Reply:
[ci(compatible-test): push-to-main event is not necessary if using merge queue.](https://github.com/axonweb3/axon/pull/1392/commits/723e52eca565d4aea80cfeceb0ebdbbb08918156)